### ISSUE TITLE
`rebar3 hex docs --revert 0.8.0` does nothing

### ISSUE THREAD

**Issue by lpil** [erlef/rebar3_hex#165]
### Environment ###

Appologies, my disappointing Windows machine is being unreliable so I can't copy/paste logs from my terminal.

### Current behaviour ###

When I run `rebar3 hex docs --revert $VERSION` it prints that edoc is being run and then stops. The documentation is not removed

### Expected behaviour ###

Documentation should be removed from hexdocs

**Comment by starbelly** [comment]
@lpil Hey Louis, are you running the latest version of rebar3_hex? I will try to replicate this tomorrow.

**Comment by starbelly** [comment]
I can not replicate this. `--revert` works as expected in that the docs are deleted for a specific version. Tested on hex.pm itself. Screen shot below. The 0.10-rc425 release had docs, then I ran `rebar3 hex docs --revert 0.1.0-rc425` and it deleted the docs. 


```
rebar3 hex docs --revert 0.1.0-rc425
===> Verifying dependencies...
===> Compiling jb_private123
===> Running edoc for jb_private123
Local Password:
===> Successfully deleted docs for jb_private123 0.1.0-rc425
```

<img width="799" alt="Screen Shot 2020-05-01 at 7 51 24 PM" src="https://user-images.githubusercontent.com/39971740/80851156-7ed6cf00-8be5-11ea-8cd6-8bc7be09ff7d.png">


Likewise and for good measure, let's do that for 0.1.0-rc424

```
rebar3 hex docs --revert 0.1.0-rc424
===> Verifying dependencies...
===> Compiling jb_private123
===> Running edoc for jb_private123
Local Password:
===> Successfully deleted docs for jb_private123 0.1.0-rc424
```

<img width="673" alt="Screen Shot 2020-05-01 at 7 54 14 PM" src="https://user-images.githubusercontent.com/39971740/80851185-b34a8b00-8be5-11ea-9bed-335a026899ac.png">


Note that I was using the latest version of rebar3_hex.
